client/app/bundles/course/assessment/skills/components/tables/SkillsTable.tsx

```diff
@@ -198,12 +199,13 @@ const SkillsTable: FC<Props> = (props: Props) => {
         customBodyRenderLite: (dataIndex): JSX.Element | string => {
           if (tableType === TableEnum.SkillBranches) {
             return (
-              <span
-                className="fa fa-chevron-right"
+              <IconButton
```

Comment:
Not sure why you used button here

Reply:
Fixed by me

Reply:
I'm sorry it's just my poor choice of implementation. You're right, the new implementation way is more compact and easier to read. will give more thought regarding this next time.